pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConsumerBuilderImpl.java

```diff
@@ -60,12 +59,18 @@
         this(client, new ConsumerConfigurationData<T>(), schema);
     }
 
-    private ConsumerBuilderImpl(PulsarClientImpl client, ConsumerConfigurationData<T> conf, Schema<T> schema) {
+    ConsumerBuilderImpl(PulsarClientImpl client, ConsumerConfigurationData<T> conf, Schema<T> schema) {
         this.client = client;
         this.conf = conf;
         this.schema = schema;
     }
 
+    @Override
+    public ConsumerBuilder<T> loadConf(Map<String, Object> config) {
+        this.conf = ConfigurationDataUtils.loadData(config, ConsumerConfigurationData.class);
```

Comment:
Instead of rewriting from scratch. Would it be possible to override the current `conf` object? This would make make intuitive things like: 
`builder.serviceUrl("xxx").loadConf(myMap)`

Maybe it's not super-useful, but it's not evident to me that the above line will disregard the earlier `serviceUrl()` call.